### ISSUE TITLE
Build failure with XCode 12.3

### ISSUE THREAD

**Issue by pfumagalli** [NativeScript/NativeScript#9098]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 7.0.11
 - Cross-platform modules: 7.0.13
 - Android Runtime: 7.0.1
 - iOS Runtime: 7.0.3
 - XCode Version: **12.3**
 - Plugin(s): _none_

**Describe the bug**
I just updated to XCode 12.3 (auto-update) and now builds (for any iOS target) are failing with a cryptic:

`error: Building for iOS Simulator, but the linked and embedded framework 'TNSWidgets.framework' was built for iOS + iOS Simulator. (in target '...' from project '...')`

or

`error: Building for iOS, but the linked and embedded framework 'TNSWidgets.framework' was built for iOS + iOS Simulator. (in target '...' from project '...')`

**To Reproduce**

A simple `tns build ios` will do...

Just to make sure, I restarted from scratch:

```
$ tns create 'BadXcode' --ts --path 'badxcode' --appid 'badxcode'
...
$ cd ./badxcode/BadXcode/
$ npm install
...
$ $ tns build ios
Copying template files...
⠙ + @nativescript/ios@7.0.6
...
Xcode build...
note: Using new build system
note: Building targets in parallel
note: Using codesigning identity override: 
note: Planning build
note: Constructing build description
warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.3.99. (in target 'MDFInternationalization' from project 'Pods')
error: Building for iOS Simulator, but the linked and embedded framework 'TNSWidgets.framework' was built for iOS + iOS Simulator. (in target 'BadXcode' from project 'BadXcode')
** BUILD FAILED **

Command xcodebuild failed with exit code 65
$
```

**Expected behavior**

It should build :-)

**Additional context**

Attaching the `--log trace` output... Ultimately the culprit is

```
/Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild \
  ONLY_ACTIVE_ARCH=NO \
  CODE_SIGN_IDENTITY= build \
  -configuration Debug \
  -sdk iphonesimulator \
  BUILD_DIR=/Users/pier/Developer/badxcode/BadXcode/platforms/ios/build \
  SHARED_PRECOMPS_DIR=/Users/pier/Developer/badxcode/BadXcode/platforms/ios/build/sharedpch \
  -workspace /Users/pier/Developer/badxcode/BadXcode/platforms/ios/BadXcode.xcworkspace \
  -scheme BadXcode
```

[trace.txt](https://github.com/NativeScript/NativeScript/files/5696059/trace.txt)

**Comment by pfumagalli** [comment]
In the process of rolling back to XCode 12.2...

**Comment by pfumagalli** [comment]
I can confirm that by rolling back to XCode 12.2 everything just works fine...

```
$ tns build ios
Preparing project...
...
Xcode build...
...
Project successfully built.
The build result is located at: /Users/pier/Developer/badxcode/BadXcode/platforms/ios/build/Debug-iphonesimulator/BadXcode.app
$ /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -version
Xcode 12.2
Build version 12B45b
$
```

**Comment by pfumagalli** [comment]
Duplicate of #9097 ... Sorry, didn't see the issue
